pom.xml

```diff
@@ -143,6 +149,81 @@
 
   <build>
     <plugins>
+      <!--
```

Comment:
this is not needed by clm:evaluate

Reply:
Yes, I had used it with iq-cli. Removing now.